src/test/java/vention/pagesTest/BookFlowTest.java

```diff
@@ -0,0 +1,54 @@
+package vention.pagesTest;
+
+import vention.driver.DriverManager;
+import vention.pages.CookiePopup;
+import vention.pages.HomePage;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import org.testng.Assert;
+
+public class BookFlowTest extends BaseTest {
+  private static final String FARE_TYPE_ONE_WAY = "oneWay";
+  private static final String FROM_CITY_NAME = "New York";
+  private static final String FROM_CITY_CODE = "new-york,-john-f-kennedy-(ny)-(jfk),-usa";
+  private static final String TO_CITY_NAME = "London";
+  private static final String TO_CITY_CODE = "london,-gatwick-(lgw),-united-kingdom";
+  private static final String DEPARTURE_DATE = "calendar-day-30/06/2025";
+  private static final String HOME_URL = "/home";
+
+  private CookiePopup cookiePopup;
+  private HomePage homePage;
+
+  @Test
+  public void testBookFlight() {
+    homePage = new HomePage();
+    homePage.openPage();
+
+    final String actualUrl = DriverManager.getDriver().getCurrentUrl();
+    Assert.assertTrue(actualUrl.contains(HOME_URL), "User should be in home page");
+
+    cookiePopup = new CookiePopup();
+    Assert.assertTrue(cookiePopup.isRejectAllButtonDisplayed(), "Cookie popup should be visible during pre-login page with a reject all button");
```

Comment:
Is rejecting cookies part of the test? why do we need to fail the test if this button is not displayed? 

Shouldn't we reject them if this button is present, if not - we can proceed anyway